lib/utils/runOpen.js

```diff
@@ -13,16 +13,22 @@ function runOpen(uri, options, log) {
     openMessage += `: ${options.open}`;
   }
 
-  const pageUrl =
-    options.openPage && isAbsoluteUrl(options.openPage)
-      ? options.openPage
-      : `${uri}${options.openPage || ''}`;
+  const pages =
+    typeof options.openPage === 'string'
+      ? [options.openPage]
+      : options.openPage || [''];
 
-  return open(pageUrl, openOptions).catch(() => {
-    log.warn(
-      `${openMessage}. If you are running in a headless environment, please do not use the --open flag`
-    );
-  });
+  return Promise.all(
+    pages.map((page) => {
+      const pageUrl = page && isAbsoluteUrl(page) ? page : `${uri}${page}`;
+
+      return open(pageUrl, openOptions).catch(() => {
+        log.warn(
+          `${openMessage}. If you are running in a headless environment, please do not use the --open flag`
```

Comment:
I think this log is very noisy when logs are many. What do you think?

Reply:
I think it is normal, something can be happens and we should output warning about this

Reply:
Outputted messages are always the same. So it is better to put the output together or add a specific word to the message.

Reply:
Yes, you are right, let's insert open page in error message

Reply:
@hiroppy @evilebottnawi I have made adjustments. Is this what you had in mind?